docs/components/modules/ROOT/pages/hwcloud-iam-component.adoc

```diff
@@ -1,7 +1,7 @@
 [[hwcloud-iam-component]]
 = Huawei Identity and Access Management (IAM) Component
 //THIS FILE IS COPIED: EDIT THE SOURCE FILE:
-:page-source: components/camel-huawei/camel-huaweicloud-iam/src/main/docs/hwcloud-iam-component.adoc
+:page-source: componentscamel-huaweicamel-huaweicloud-iamsrcmaindocshwcloud-iam-component.adoc
```

Comment:
This is wrong, because you're on Windows I guess

Reply:
Have run the mvn routines from a mac/linux platform. Should look good  now